src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -51,7 +54,7 @@ export class Main extends Component {
 
   componentDidUpdate() {
     const { currentlyLoggedIn, user } = this.props;
-    const { mhvTransitionEligible } = user || {};
+    const { mhvTransitionEligible, mhvTransitionComplete } = user || {};
```

Comment:
If this is going to live on the user model, we can expect that it will always be truthy post-transition. We will need to find a way to ensure the user only sees this modal once. Session storage does not seem suited for this as a user could re-see the modal by swapping devices or clearing storage.

Not sure what backend plans are but maybe we could get this as a param in the auth callback that is only given the first time. This would rid FE of the need to track when/if it was viewed.

Reply:
this is also non-blocking as we have no backend to work with currently

Reply:
I'm down for either a query parameter or something in the User Model.